libraries/AP_NavEKF2/AP_NavEKF2_Outputs.cpp

```diff
@@ -541,6 +548,9 @@ void NavEKF2_core::send_status_report(mavlink_channel_t chan)
     if (filt_state.flags.pred_horiz_pos_abs) {
         flags |= EKF_PRED_POS_HORIZ_ABS;
     }
+    if (filt_state.flags.gps_glitching) {
+        flags |= (1<<15);
```

Comment:
Not sure how to do that. I don't want to delay getting this in waiting to get a change to mavlink pushed through.